README.de.md

```diff
@@ -0,0 +1,88 @@
+# __10500__ color-names 🎨
+[![GitHub release](https://img.shields.io/github/release/meodai/color-names.svg)](https://github.com/meodai/color-names/)
+[![npm version](https://img.shields.io/npm/v/color-name-list.svg)](https://www.npmjs.com/package/color-name-list)
+[![npm](https://img.shields.io/npm/dt/color-name-list.svg)](https://www.npmjs.com/package/color-name-list)
+[![Travis](https://img.shields.io/travis/meodai/color-names.svg)](https://travis-ci.org/meodai/color-names)
+[![license](https://img.shields.io/npm/l/color-name-list.svg?colorB=ff77b4)](https://github.com/meodai/color-names/blob/master/LICENSE)
+[![color count](https://img.shields.io/badge/__10500__-colors-orange.svg)](https://github.com/meodai/color-names/blob/master/src/colornames.csv)
+
+Ein handverlesene Liste von __10500__ eindeutiger Farbname von verschiedene Quellen und tausende von Einreichungen. [Versuch es selber](http://codepen.io/meodai/full/mEvZRx/) or read [the full list](https://docs.google.com/spreadsheets/d/14ny2oB7g5Tof9TmKiaaDFv25XSCRt-LlBRJhIDz_3Mo/pubhtml?gid=40578722)
+
+> Die Namen der Farbe funktionieren wie ein Faden an einer schrecklich schlanken Nadel befestigt, fähig unsere empfindlichsten Gefühle und Erinnerungen zusammenzutragen. Wann die Nadel auf das Ziel trifft, fühlen wir uns Lust oder Empathie. **Kenya Hara – White**
+
+## Über 📋
+
+Ziel dieses Projektes ist es, eine möglichst große Liste von Farbnamen zu erstellen. Wir haben verschiedene [Liste](#sources-) fusioniert, die Namen geändert, wenn es Duplikate mit verschiedenen HEX-Werten gab, und verschob die Farben ein bißchen wann es gibt identische farben mit andere namen.
```

Comment:
Ziel dieses Projektes ist es, eine  möglichst **vollständige** (Grosse is not Wrong, but I rather say complete here) ...: Wir haben verschiedene Liste**n** **zusammengetragen** (fusioniert is correct, but no-one would say that)... ...und wir haben die **Farben ein wenig verändert wenn die selbe Farbe verschiedene Namen trug** (In German you are "carrying a name")